### ISSUE TITLE
accessibility: add aria-label for VoiceOver clearInput function in ion-input

### ISSUE THREAD

**Issue by terrancle** [ionic-team/ionic#21537]
@5.2.1

Hello,

When the clearInput attribute is used in an ion-input, the button used to empty the cache is not labeled for VoiceOver on iOS.
VoiceOver says "button" without fully understanding the purpose of the button.
Can you add a "Clear" aria-label to this button?
For translations into several languages, it would even be good to allow developers to customize the text of the aria-label for clearInput;)
Thank you

**Comment by liamdebeasi** [comment]
Thanks for the issue. This has been resolved via https://github.com/ionic-team/ionic/pull/21538, and a fix will be available in the next release of Ionic Framework.

In future versions of the framework, we hope to make it possible to let users customize these labels so they can be translated.
